roles/ansible-keylime-tpm20/tasks/keylime.yml

```diff
@@ -15,6 +15,11 @@
     mode: 0644
     remote_src: yes
 
+- name: Convert keylime.conf to new config format
+  shell: "/usr/bin/python3 convert_config.py && cp agent.conf verifier.conf tenant.conf registrar.conf ca.conf logging.conf /etc/keylime/"
```

Comment:
Does it need to create the /etc/keylime directory first?

Reply:
Oops, good catch. Adding that now.